lib/facade/MonitoringFacade.ts

```diff
@@ -605,6 +607,12 @@ export class MonitoringFacade extends MonitoringScope {
     return this;
   }
 
+  monitorSyntheticsCanary(props: SyntheticsCanaryMonitoringProps) {
+    const segment = new SyntheticsCanaryMonitoring(this, props);
+    this.addSegment(segment, props);
+    return this;
+  }
```

Comment:
nitpick: Do you think we should prefix it with `cloudwatch`? That is `monitorCloudwatchSyntheticsCanary`?

Reply:
I just noticed that this is under its own `synthetics` module. Disregard this comment